test/known_issues/test-stdout-buffer-flush-on-exit.js

```diff
@@ -0,0 +1,19 @@
+'use strict';
+// Refs: https://github.com/nodejs/node/issues/2148
+require('../common');
+const assert = require('assert');
+const execSync = require('child_process').execSync;
+
+const longLine = 'foo bar baz quux quuz aaa bbb ccc'.repeat(80);
+const expectedLength = (longLine.length * 999) + 1;
```

Comment:
Probably. Good point. Maybe the thing to do is compare `stdout.trim().length` in the assertion so as to avoid any assumptions whatsoever about the line separator?